culture/review_culture.md

```diff
@@ -0,0 +1,142 @@
+think are harmful, or approve a change over others' objections. Someone can be
+a maintainer within a specific domain: for example, acting as maintainer for a
+particular repository, or even maintaining a particular guide. Since
+maintainers have special powers, they also have a special responsibility to act
+thoughtfully and fairly. New maintainers are chosen by the existing
+maintainers. In general, people who have a long track record of acting as
+contributors and reviewers may be good candidates for becoming maintainers.
+
+## Lifecycle of a Review
+
+### Proposal
+The first step is for a contributor to propose a _change_.
+The contributor needs to do two things at this stage:
+- Flesh out the change (e.g. writing the code), so that there is something to review
+- Explain the change (e.g. writing a pull request description), so that reviewers can easily understand it
+
+Both steps are vital, and contributors should put real effort into explaining the change well.
+Its not fair to ask a reviewer to puzzle through an undocumented change.
+
+Finally, before requesting a review, the contributor should _self-review_ their change.
+It's amazing how many issues can be spotted and fixed during self-review.
+This is respectful of reviewers' time, and makes the process a lot smoother.
+
+### Review
+
+Once a change is ready and has been self-reviewed, it's ready for review. Since
+most SourceCred artifacts are tracked via repositories on GitHub, this will
+usually mean creating a pull request.
+
+Once the change is proposed, anyone in the project can add their own review.
+The person proposing the change can also request specific reviewers, if certain
+individiuals have context or knowledge thet will make their reivew particularly
+valuable.
+
+On finishing their review, the reviewer can:
+- ask questions to better understand the change
+- suggest changes to it
+- approve the change
+
+Most changes go through a few rounds of reviews before being approved.
+
+### Approval and Merging
+
+During the review process, a successful change will accumulate "approvals" from
+satisfied reviewers. At some point, the change will have "enough approvals" and
+will be ready to merge. There's no hard-and-fast rule to determine when a
+change has enough approvals. Generally, simple and small changes are easy to
+approve, whereas changes that are breaking new ground, or have wide-ranging
+consequences, will require more reviews. If you're unsure as to whether a
+change is ready to merge, ask one of the maintainers.
+
+### Changes that never merge
+
+Some changes don't ever merge. A few good reasons why this may happen include:
+- Reviewers may find that the change isn't actually needed, is basically
+  unsound, or isn't aligned with the project's values or goals
+- Over the course of review, the contributor or reviewers may find a different
+  and better way to achieve the same goals
+
+# Best Practices for Reviewers
```

Comment:
Suggest adding a section on “nit-and-approve” as an encouraged practice,
calling out the trust considerations. (Or, I can draft such an addition
and submit my own PR, if you’d prefer?)

I have a couple of other best practices for proposers and reviewers in
my mental model, and will try to think about which ones are worth
surfacing in a document like this.

Reply:
Would love to have a section on nit-and-approve. Please draft the addition as a separate PR.

Reply:
Sure: will do.